io.openems.edge.controller.ess.timeofusetariff/src/io/openems/edge/controller/ess/timeofusetariff/ControlMode.java

```diff
@@ -0,0 +1,31 @@
+package io.openems.edge.controller.ess.timeofusetariff;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum ControlMode implements OptionsEnum {
```

Comment:
Why is this a `OptionsEnum`? For config settings a normal enum is better suitable,